Expenses.js

```diff
@@ -0,0 +1,43 @@
+var React = require('react-native');
+var {
+  StyleSheet,
+  Image,
+  View,
+  TouchableHighlight,
+  ListView,
+  Text,
+  Component
+} = React;
+
+class Expenses extends Component {
+
+  constructor(props) {
+    super(props);
+    var dataSource = new ListView.DataSource(
+      {rowHasChanged: (r1, r2) => r1[0] !== r2[0]});
+    this.state = {
+      dataSource: dataSource.cloneWithRows(this.props.listings)
+    };
+  }
+
+  renderRow(rowData, sectionID, rowID) {
+    return (
+      <TouchableHighlight
+          underlayColor='#dddddd'>
```

Comment:
@rrgayhart yep I'll move the inline styling to the proper style regions. The React Native tutorials I've seen all have style objects in-file that are referenced. Still haven't figured out if this is standard or sloppy.